<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_Dialog</class>
 <widget class="QDialog" name="Add_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSpinBox" name="addQuantity_spinBox">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>20</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>123</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="revocation_pushButton">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>123</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
  </widget>
  <widget class="QPushButton" name="done_pushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>123</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>完成</string>
   </property>
  </widget>
  <widget class="QLabel" name="remind_label">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>80</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_background">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>0</y>
     <width>251</width>
     <height>169</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_line_1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>126</y>
     <width>5</width>
     <height>44</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1149835.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_line_2">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>125</y>
     <width>5</width>
     <height>43</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1149835.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_line_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>164</y>
     <width>251</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1149835 - 副本.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_background</zorder>
  <zorder>addQuantity_spinBox</zorder>
  <zorder>confirm_pushButton</zorder>
  <zorder>revocation_pushButton</zorder>
  <zorder>done_pushButton</zorder>
  <zorder>remind_label</zorder>
  <zorder>label_line_1</zorder>
  <zorder>label_line_2</zorder>
  <zorder>label_line_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
